pkg/monitor/agent/listener/listener.go

```diff
@@ -17,9 +17,9 @@ package listener
 import (
 	"net"
 	"os"
-	"syscall"
 
 	"github.com/cilium/cilium/pkg/monitor/payload"
+	"golang.org/x/sys/unix"
```

Comment:
Add an empty line before this import.

Reply:
I'll clean all these imports up after we get swagger and master fixed. https://github.com/cilium/cilium/pull/10336